dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/DatadogCoreTest.kt

```diff
@@ -803,6 +805,20 @@ internal class DatadogCoreTest {
         assertThat(testedCore.isDeveloperModeEnabled).isFalse()
     }
 
+    @RepeatedTest(128)
```

Comment:
Shouldn't it be just `@Test` here? Is it needed to repeat it so many times?

Reply:
You're right, i used it locally :D